### PR TITLE
Vscode debug support

### PR DESCRIPTION
**Pull request by coinzdude**
This is a "rough, appears to be working" set of changes to allow VS code to set breakpoints in the source TS files and hit them as bound breakpoints.

It does not consider breaking these out for debug vs release.  Also, I'm not certain of which of these was the significant change, if not all of them.

There is:

1. a .vscode\launch.json  in the top level folder
2. packages/selenium-ide/tsconfig.json with some flags to instruct the compiler to embed the 
3. a new dependency for source-map-support
4. a hard coded debugger line that I forgot to remove

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,7 +54,8 @@
     "node-fetch": "^3.2.3",
     "selenium-webdriver": "^4.1.1",
     "tar": "^6.1.11",
-    "unzipper": "^0.10.11"
+    "unzipper": "^0.10.11",
+    "source-map-support": "^0.5.21"
```
Comment (toddtarsi) [inline package.json]: @coinzdude - Super minor, but can you move this to an alphabetical order?

File: packages/selenium-ide/src/main/session/controllers/Driver/start.ts [hunk 2]
```diff
@@ -46,7 +46,7 @@ const startDriver: StartDriver =
                 'node_modules',
                 'electron-chromedriver',
                 'bin',
-                'chromedriver'
+                'chromedriver.exe'
```
Comment (toddtarsi) [inline packages/selenium-ide/src/main/session/controllers/Driver/start.ts]: @coinzdude - I think this line should be this:

```
  `chromedriver${platform.os() === 'win32' ? '.exe' : ''}`
```

File: packages/selenium-ide/tsconfig.json [hunk 3]
```diff
@@ -38,4 +54,4 @@
       "path": "../side-runtime"
     }
   ]
-}
+}
```
Comment (toddtarsi) [inline packages/selenium-ide/tsconfig.json]: @coinzdude - Nice work prettifying this JSON. Can you make sure there is a newline at the end of this file? I don't know why, but the linter complains about it.

File: packages/selenium-ide/package.json [hunk 4]
```diff
@@ -13,8 +13,8 @@
     "build": "run-s build:webpack build:electron",
     "build:electron:import": "tsc --build ./electron-import",
     "build:electron": "electron-builder -c.electronVersion $(node -e \"console.log(require('electron/package.json').version)\") --publish never",
-    "build:webpack": "webpack",
-    "watch": "webpack --watch"
+    "build:webpack": "webpack --mode=development",
```
Comment (toddtarsi) [inline packages/selenium-ide/package.json:16]: At some point, we might only leave this flag hardcoded on the watch command as we get closer to having a production release process, but for our current phase, this is good.

File: .vscode/launch.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+            "windows": {
+                "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/electron"
+            },
+            "args": [
+                ".\\packages\\selenium-ide\\dist\\main-bundle.js"
+            ],
+            "outputCapture": "std"
+        }
+    ]
+}
```
Comment (toddtarsi) [inline .vscode/launch.json:21]: @coinzdude - Super minor, please add a newline here at the end

**Review by toddtarsi — COMMENTED** [review]
Okay, I added a few comments for tweaks. All and all, this looks really good, and I'm so excited to have first class IDE support in vscode. Thanks and great work!!!

File: package.json (hunk 1 above)
Comment (toddtarsi) [reply package.json]: Actually, I need to go one further! Can you move this down into the `packages/selenium-ide/package.json` file and alphabetize it there. This is a devDependency btw.

Comment (toddtarsi) [reply package.json]: @coinzdude - Nice work placing this elsewhere

File: packages/selenium-ide/src/main/session/controllers/Driver/start.ts (hunk 2 above)
Comment (coinzdude) [reply packages/selenium-ide/src/main/session/controllers/Driver/start.ts]: I have some edits I need to get into a new PR or revise this one. Not sure

File: packages/selenium-ide/tsconfig.json (hunk 3 above)
Comment (coinzdude) [reply packages/selenium-ide/tsconfig.json]: I was able to get blank lines at the end by changing VS code options   
```
"editor.formatOnSave": false,
```

**Comment by coinzdude** [comment]
I hope I am supposed to close this now that I've submitted edits.

**Review by toddtarsi — APPROVED** [review]
LGTM 👍

Thanks for the hard work on this!

**Pull request merged**